### ISSUE TITLE
Cannot be used with Bootstrap 4

### ISSUE THREAD

**Issue by alvarosaburido** [Innologica/vue2-daterange-picker#19]
The `.dropdown-menu` class inside the component is too generic, the Popover with the calendar will stay with `display:none`. I suggest to change the classes inside.

**Comment by nkostadinov** [comment]
I will soon do this because I need it in a project with BS4. I will inform you here when done.

**Comment by nkostadinov** [comment]
should look good on bs4 now

**Comment by nkostadinov** [comment]
Yes, I noticed that it doesn't show up. I found out that I had included both css for BS3 and BS4 while testing.  Will reopen and fix. sorry :)

**Comment by nkostadinov** [comment]
updated component and demo to use BS4.

 you should not include the JS file of the datepicker.
